integration/monai/examples/spleen_ct_segmentation_sim/README.md

```diff
@@ -78,7 +78,7 @@ We can also specify the client names via the `--clients` argument
 and assign them to the appropriate GPU device using the `--gpu` argument.
 
 ```
-nvflare simulator job --workspace /tmp/nvflare/sim_spleen_ct_seg --threads 2 --clients site-1,site-2 --gpu 0,1
+nvflare simulator job --workspace /tmp/nvflare/sim_spleen_ct_seg --clients site-1,site-2 --gpu 0,1
```

Comment:
this `--threads 2` should keep?

Reply:
The gpu flag will already create two threads. It's redundant.